dev-docs/source/GettingStarted.rst

```diff
@@ -52,24 +52,14 @@ Unfortunately CMake can't find it out of the box and the following steps are req
 * create a new string value named ``InstallPath`` within this key and set the value
   to point to the install directory of Graphviz.
 
+Windows Subsystems for Linux (WSL2)
```

Comment:
Thanks.
Really picky thing I just spotted in a couple of places. It's the *Windows Subsystem for Linux* without the **s** on subsystem. If we can correct that in the various places below too then this is ready to go!